.github/workflows/ci-benchmark.yaml

```diff
@@ -17,7 +18,8 @@ jobs:
       github.event_name == 'schedule' ||
       (
         github.event.label.name == 'run_benchmark' &&
-        github.event.name != 'pull_request_target'
+        github.event.name != 'pull_request_target' &&
+        github.event.schedule != '0 0 * * *'
```

Comment:
Are we sure with this that the Benchmark job will keep continue running on Monday at 00:00. Because, I think with this it may never run.

Reply:
earlier check condition was not right, please let me know I have updated the check